Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
btw, are there any advantages of the espressif bootloader? Would it make sense to to support rboot only to simplify the framework?